playbooks/compute/nto/roles/configurecluster/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+artifacts_folder: /artifacts
+ignore_cgroups_version: "true"
+rt_kernel: false
+must_gather_dir: "/tmp/mustgather11"
```

Comment:
Can you please improve the naming here? /tmp/mustgather

Reply:
thanks